### PR TITLE
Remove out of date options

### PR DESCRIPTION
**Pull request by CaptainAwesomeDi**
Remove: Automated Test should be expected by reviewer by default
Remove: Screenshot should be with description
Remove: We no longer uses staging environment
Remove: Assignee because there is already an option for contact CSS in other

## Related issues

- [KEY-123](https://netengine-trial.atlassian.net/browse/:id)
- [Bugsnag error](https://app.bugsnag.com/netengine/scout-recruit/errors/:id)

## Type of change

- [ ] Request (code for a specific purpose that will not add a feature. e.g. one off Rake task)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance task (Rake tasks or changes that do not change logic that will affect the application behaviour)

## Checklist

- Manual tests
  - [ ] Changes were tested locally by the author
  - [ ] Changes were tested on staging by the author
  - [ ] Changes were tested on staging by reviewers and reviewers mentioned themselves in the "Assignee" section above
  - [ ] Changes were not tested and the reason was explained in the "How to test" section above
- Automated tests
  - [ ] Tests were written where appropriate
- Labels
  - [ ] `staging` label was added after this PR was released to staging
- Others
  - [ ] There are no typos, spelling, or grammatical errors in UI/UX text
  - [ ] Client Success Team (for Scout) or client (for Agency) was notified before deployment if this PR involves risky changes; e.g.: running Rake tasks for modifying or deleting resources

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -33,25 +34,11 @@
...
-  - [ ] Changes were tested on staging by the author
-  - [ ] Changes were tested on staging by reviewers and reviewers mentioned themselves in the "Assignee" section above
   - [ ] Changes were not tested and the reason was explained in the "How to test" section above
-- Automated tests
-  - [ ] Tests were written where appropriate
-- Labels
-  - [ ] `staging` label was added after this PR was released to staging
+
 - Others
-  - [ ] There are no typos, spelling, or grammatical errors in UI/UX text
```
Comment (jessicams91) [inline pull_request_template.md:56]: Hi @CaptainAwesomeDi the grammatical error and automated test checkboxes as not used a lot but I believe they serve as a reminder

File: pull_request_template.md [hunk 2]
```diff
@@ -33,25 +34,11 @@
...
-
-## Screenshots
-
-<!-- Add screenshots, link to Loom videos, or link to wireframes/mockups if they are relevant; e.g.: for frontend work -->
-
 ## Checklist
 
 - Manual tests
   - [ ] Changes were tested locally by the author
-  - [ ] Changes were tested on staging by the author
```
Comment (jessicams91) [inline pull_request_template.md:48]: It still should say UAT here for recruit and staging for onboard, so depends on which repos will use this

File: pull_request_template.md (hunk 1 above)
Comment (CaptainAwesomeDi) [reply pull_request_template.md:56]: I believe typos can be avoided by using spellcheck plugin, automate tests shouldn't be a reminder but a habit

File: pull_request_template.md (hunk 2 above)
Comment (CaptainAwesomeDi) [reply pull_request_template.md:48]: recruit is automatically overwritten by it's repos pull request template. onboard also has it's own PR template

**Review by jessicams91 — APPROVED** [review]

**Pull request merged**